internal/config/config_doc.go

```diff
@@ -242,8 +242,8 @@ func init() {
 	AzureConfigDoc.Fields[6].Name = "clientSecretValue"
 	AzureConfigDoc.Fields[6].Type = "string"
 	AzureConfigDoc.Fields[6].Note = ""
-	AzureConfigDoc.Fields[6].Description = "Client secret value of the Active Directory app registration credentials."
-	AzureConfigDoc.Fields[6].Comments[encoder.LineComment] = "Client secret value of the Active Directory app registration credentials."
+	AzureConfigDoc.Fields[6].Description = "Client secret value of the Active Directory app registration credentials. Alternative use CONSTELL_AZURE_CLIENT_SECRET_VALUE environment variable."
+	AzureConfigDoc.Fields[6].Comments[encoder.LineComment] = "Client secret value of the Active Directory app registration credentials. Alternative use CONSTELL_AZURE_CLIENT_SECRET_VALUE environment variable."
```

Comment:
don't forget to regenerate before merge

Reply:
Good point! Looking forward to @3u13r 's check there 😉